version-control/tutorial/README.md

```diff
@@ -0,0 +1,13 @@
+# Super Sandwich Survey
+
+Sandwiches are a staple part of many people’s diets, particularly in western culture. However, little data exists on sandwich eating habits today. This repository will be used as a reference guide for anyone taking part in the super sandwich survey. 
+
+Thank you for taking part in the super sandwich survey. Your responses will contribute to a larger resource on people’s experiences with sandwiches. Please read the instructions file to learn more information about how to interact with the sandwich survey and how to log a new sandwich that is not currently in the sandwich database.
+
+---
+
+- It is strongly encouraged to have a README.md file in every git repository. A README.md file gives an overview of what the project it is, and (when on github.com) how other people can get involved, and who maintains and contributes to the project. We'll learn more about these best practices in lessons 2 & 3. 
+
+- The md stands for markdown. Markdown is a language that allows you to add formatting elements to plain text documents. When it is rendered (normally on github.com, but also possible to do locally) you will see how the formatting elements change how our block of text appears. We have a few in this document: '#' will make this line a heading, '---' draws a horizontal line, and '-' at the start of a line is a bullet point. Below, we have used the '**' to make text bold.
```

Comment:
I would also remove this. If they need to make any changes to the formatting, we can explain that verbally, but I would suggest keeping the changes to text so we don't have to get into markdown.

Reply:
Sounds good! All of the changes I had planned where content not formatting